dataprofiler/tests/profilers/test_datalabeler_options.py

```diff
@@ -0,0 +1,126 @@
+        expected_val = {'data_labeler_dirpath': None, 
+            'max_sample_size': None, 
+            'is_enabled': True,
+            'data_labeler_object': None}
+
+        self.assertDictEqual(expected_val, options.properties)    
+    
+    def test_set_helper(self):
+        super().test_set_helper()
+
+    def test_set(self):
+        super().test_set()
+    
+    def test_validate_helper(self):
+        super().test_validate_helper()
+        optpth = self.get_options_path() 
+
+        # Test valid dirpath
+        options = self.get_options()
+        options.set({'data_labeler_dirpath': ''})
+        self.assertEqual([], options._validate_helper())
+
+        # Test valid sample size
+        options = self.get_options()
+        options.set({'max_sample_size': 1})
+        self.assertEqual([], options._validate_helper()) 
+
+        # Test valid data labeler object
+        options = self.get_options()
+        options.set({'data_labeler_object': BaseDataLabeler()})
+        self.assertEqual([], options._validate_helper())
+
+        # Test invalid dirpath
+        options = self.get_options()
+        options.set({'data_labeler_dirpath': 0})
+        expected_error = "{}.data_labeler_dirpath must be a string.".format(optpth)
+        self.assertEqual([expected_error], options._validate_helper())    
+
+        # Test invalid sample size
+        options = self.get_options()
+        options.set({'max_sample_size': ''})
+        expected_error = "{}.max_sample_size must be an integer.".format(optpth)
+        self.assertEqual([expected_error], options._validate_helper())    
+        
+        # Test max sample size less than or equal to 0
+        options = self.get_options()
+        expected_error = "{}.max_sample_size must be greater than 0.".format(optpth)
+        options.set({'max_sample_size': 0})
+        self.assertEqual([expected_error], options._validate_helper())    
+        options.set({'max_sample_size': -1})
+        self.assertEqual([expected_error], options._validate_helper())    
+    
+        # Test invalid data labeler object
+        options = self.get_options()
+        expected_error = 'DataLabelerOptions.data_labeler_object must be a BaseDataLabeler object.'
+        options.set({'data_labeler_object': 0})
+        self.assertEqual([expected_error], options._validate_helper())
+
+    def test_validate(self):
+        super().test_validate()
+        optpth = self.get_options_path() 
+
+        # Test valid dirpath
+        options = self.get_options()
+        options.set({'data_labeler_dirpath': ''})
+        self.assertEqual(None, options.validate())
+
+        # Test valid sample size
+        options = self.get_options()
+        options.set({'max_sample_size': 1})
+        self.assertEqual(None, options.validate()) 
+
+        # Test valid sample size
+        options = self.get_options()
+        options.set({'max_sample_size': 1})
+        self.assertEqual(None, options.validate()) 
```

Comment:
Copied the same test case twice, and while I'm here I noticed you don't have a test for valid DataLabeler object into validate

Reply:
My bad, I meant to add a test for valid DataLabeler and must have added a valid sample size test - fixed in new commit!